rele/config.py

```diff
@@ -83,6 +83,20 @@ def setup(setting=None, **kwargs):
     return config
 
 
+def subscriptions_have_same_memory_address(a_subscription, another_subscription):
```

Comment:
```suggestion
def is_same_susbscription(a_subscription, another_subscription):
```
I would include the `is` verb in the method naming